tests/integration/targets/mso_schema_template_anp_epg_selector/tasks/main.yml

```diff
@@ -354,9 +354,6 @@
     epg: ansible_test_1
     selector: selector_2
     expressions:
-      - type: expression_1
-        operator: in
-        value: test
```

Comment:
Not sure why we removed this block. I understand the type should be different but we should still have a case with more than 1 expression in one go.